tests/static/saveToCase/close_property.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<h:html xmlns:h="http://www.w3.org/1999/xhtml" xmlns:orx="http://openrosa.org/jr/xforms" xmlns="http://www.w3.org/2002/xforms" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:jr="http://openrosa.org/javarosa" xmlns:vellum="http://commcarehq.org/xforms/vellum">
+	<h:head>
+		<h:title>Untitled Form</h:title>
+		<model>
+			<instance>
+				<data xmlns:jrm="http://dev.commcarehq.org/jr/xforms" xmlns="http://openrosa.org/formdesigner/E1B03643-109D-4DC6-9737-815E4401DE79" uiVersion="1" version="1" name="Untitled Form">
+					<name />
+					<save_to_case vellum:role="SaveToCase">
+						<c:case xmlns:c="http://commcarehq.org/case/transaction/v2" case_id="" date_modified="" user_id="">
```

Comment:
Is there any semantic meaning to the `<save_to_case vellum:role="SaveToCase">` element, or is it just a placeholder? I would prefer not to add extra levels to the form hierarchy if we don't need to. I've never seen a hand-written form that has a `<case>` block inside of a container like this, but that doesn't mean it's never done that way.

I would prefer to do this the way the balance and transfer widgets do it. That is, they match a specific tag name (in this case `case`) with a specific `xmlns` attribute value (`http://commcarehq.org/case/transaction/v2`). To do that we either need to find a unique identifier on the case block (would the `case_id` attribute work? ... might need to do some nodeID gymnastics since it appears to allow a blank value) or we need to add an arbitrary vellum-specific attribute like `vellum:id="save_to_case"`.